src/log.rs

```diff
@@ -206,7 +237,13 @@ unsafe extern "C" fn log_callback(
                 "gstpad.parent.pending_state" = gstpad_parent_pending_state_value;
             ];
             let valueset = fields.value_set(&values);
-            let event = Event::new(meta, &valueset);
+
+            let event = match user_span {
+                Some(user_span) => Event::new_child_of(user_span, meta, &valueset),
+                None => Event::new(meta, &valueset),
+            };
+            //dbg!(&event);
```

Comment:
I have a suspicion that tracing semantically expects that the parent span is entered by the user code (i.e. this code), rather than leaving it up to the subscribers to deal with (i.e. the PR to `tracing-loki`). I asked about this on Discord, lets see what they say.

Reply:
Also, please remove the commented out code here and just above.

Reply:
> I have a suspicion that tracing semantically expects that the parent span is entered by the user code (i.e. this code), rather than leaving it up to the subscribers to deal with (i.e. the PR to tracing-loki)

You mean calling `user_span.enter();` before `dispatcher.event()`?

I just tried and that did not help with the test not getting the span.

Reply:
Nice catch, thanks!

I merged your branch into mine. Do you mind if I squash it all?